### PR TITLE
Added Lancia object and car models to common.js file

### PR DESCRIPTION
**Pull request by hauze-v**
#94

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -1889,6 +1889,36 @@ const allSoloCars = {
...
+      '1977': ['hs', 'ep', 'fsp'],
+      '1978': ['hs', 'ep', 'fsp'],
+      '1979': ['hs', 'ep', 'fsp'],
+      '1980': ['hs', 'ep', 'fsp'],
+      '1981': ['hs', 'ep', 'fsp'],
+      '1982': ['hs', 'ep', 'fsp'],
+      '1983': ['hs', 'fsp']
+    },
+    'Scorpion': {
+      'all': ['hs', 'bsp']
```
Comment (Bjorn248) [inline common.js:1905]: This is in `csp`

File: common.js [hunk 2]
```diff
@@ -1889,6 +1889,36 @@ const allSoloCars = {
...
+    },
+    'Scorpion (1756 cc)': {
+      '1976': ['dp']
```
Comment (Bjorn248) [inline common.js:1908]: This should be merged with the above `Scorpion`. There was only one kind of Scorpion sold in North America, and it had the 1756 cc engine. Just add `dp` to the existing `Scorpion` and you should be good!

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Some class correction and model merging.

**Comment by hauze-v** [comment]
Thanks for your feedback. Please check if it's okay now.

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**